.github/ISSUE_TEMPLATE.md

```diff
@@ -7,9 +7,17 @@
 ## What happens:
 1.
 
-## Logs
-- Logs
+## Traceback
+[//]: # (If there is a traceback please share it in a quote! You can do this by pasting the traceback text, then highlight it and press the quote button.)
```

Comment:
You can do this by pasting the traceback text, then highlight it and press the quote button -> You can do this by pasting the traceback text, highlighting it and pressing the quote button.